tools/mfassignr/mfassignr_kmdnoise.xml

```diff
@@ -0,0 +1,53 @@
+<tool id="mfassignr_kmdnoise" name="MFAssignR KMDNoise" version="@TOOL_VERSION@+galaxy0">
+    <description>Noise level assessment using the KMDNoise.</description>
```

Comment:
```suggestion
    <description>Noise level assessment</description>
```

I guess those description can be shortend a bit. KMDNoise is already in the "name"